.github/workflows/go-build-and-test.yml

```diff
@@ -0,0 +1,25 @@
+name: build-and-test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+
+    - name: Build
+      run: go build -v ./...
```

Comment:
We don't need to build the project just now as we don't have any conformance or e2e tests that need the built binaries so I think we can drop this step and save some CPU and engineer time when submitting PRs for now.

Reply:
ACK.

Reply:
I removed the build step in commit [90f200d](https://github.com/bom-squad/protobom/pull/27/commits/90f200df3df47b38f4fb4aa29749ef4c61280827)